sections/06_discussion.md

```diff
@@ -49,6 +49,86 @@ with only a couple GPUs.*
 *Some of this is also outlined in the Categorize section.  We can decide where
 it best fits.*
 
+Efficiently scaling deep learning is challenging, and there is a high
+computational cost (e.g., time, memory, energy) associated with training neural
+networks and using them for classification. For these reasons, neural networks
+have only recently found widespread use [@tag:Schmidhuber2014_dnn_overview].
+For biologists, such problems are further complicated by the immense size of
+most biological datasets.
+
+*TODO: Perhaps a visual illustrating the changes over time in the cost of
+storage
+(i.e. hard drive space), genome sequencing, and processing power. Additionally,
```

Comment:
I agree parameter counts don't tell the whole story.  Some important advances worked well because they reduce parameter counts.  If it is hard to capture the ImageNET history without going into a lot of detail, should we drop it?